ios/RCTMGL-v10/RCTMGLOfflineModule.swift

```diff
@@ -321,26 +321,24 @@ class RCTMGLOfflineModule: RCTEventEmitter {
   func createPack(_ options: NSDictionary, resolver: @escaping RCTPromiseResolveBlock, rejecter: @escaping RCTPromiseRejectBlock) {
     DispatchQueue.main.async {
       do {
+        let metadataStr = options["metadata"] as! String
+        let metadata = try JSONSerialization.jsonObject(with: metadataStr.data(using: .utf8)!, options: []) as! [String:Any]
+        let id = metadata["name"] as! String
+        let stylePackLoadOptions = StylePackLoadOptions(glyphsRasterizationMode: .ideographsRasterizedLocally, metadata: ["tag": id])!
```

Comment:
```suggestion
        let stylePackLoadOptions = StylePackLoadOptions(glyphsRasterizationMode: .ideographsRasterizedLocally, metadata: metadata)
```

Unless there is a reason for not storing the full metadata